server/transaction_test.go

```diff
@@ -256,7 +256,7 @@ func Test_Transaction_InvalidIDGiven(t *testing.T) {
 			t.Fatal("expected", http.StatusInternalServerError, "got", w.Code)
 		}
 
-		if !strings.Contains(w.Body.String(), "invalid transaction ID: does not match") {
+		if !strings.Contains(w.Body.String(), "does not match") {
```

Comment:
Yes, it changed and we discussed it here https://github.com/giantswarm/microerror/pull/3#discussion_r129614343